packages/editor/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 5.0.1 (2018-10-22)
+## 5.0.2 (2018-10-22)
```

Comment:
Instead of changing the existing entry you have add yours as unreleased like this:

```md
## 5.0.2 (unreleased)

### Polish
- Add animated logo to preview interstitial screen.

## 5.0.1 (2018-10-22)
```

Reply:
Like this? https://github.com/WordPress/gutenberg/pull/10953/commits/e92caad4e2171db8d5e21eddc0dced75bb49b5d0

Thanks for the patience. I've only ever published one project to NPM and it was a while ago. I'll get this.

Reply:
@jasmussen No worries. 🙂Now you only have to readd the `## 5.0.1 (2018-10-22)` line and then it's good to go.